django_fido/settings.py

```diff
@@ -35,6 +35,7 @@ class DjangoFidoSettings(AppSettings):
         certificate=NestedListSetting(inner_setting=FileSetting(), default=[]),
         crl_list=NestedListSetting(inner_setting=FileSetting(), default=[]),
     ), default=None)
+    resident_key = BooleanSetting(default=False)
```

Comment:
This should also be noted in `CHANGELOG` and maybe explained in `README`.

Reply:
I actually don't know what the cause of the error, the unittests went fine

Reply:
I thought I'd leave it changelog untouched until a release is ready, I can add some stuff in README

Reply:
Run `tox -equality`, there is an extra line in one of the files.

We have an `Unreleased` section in `CHANGELOG` for exactly this.

Reply:
Alright, I will have a look tomorrow

Reply:
Fixed quality check, updated changelog and readme